test/functional/specs/Personalization/C28756.js

```diff
@@ -0,0 +1,87 @@
+import { t, ClientFunction } from "testcafe";
+import createNetworkLogger from "../../helpers/networkLogger";
+import { responseStatus } from "../../helpers/assertions/index";
+import fixtureFactory from "../../helpers/fixtureFactory";
+import configureAlloyInstance from "../../helpers/configureAlloyInstance";
+import {
+  compose,
+  orgMainConfigMain,
+  debugEnabled
+} from "../../helpers/constants/configParts";
+import getResponseBody from "../../helpers/networkLogger/getResponseBody";
+import createResponse from "../../../../src/core/createResponse";
+
+const networkLogger = createNetworkLogger();
+const config = compose(
```

Comment:
I think you can use constants/debugEnabledConfig.js here instead of composing one.

Reply:
thank you!